src/Configuration/src/ConfigServerBase/ConfigServerConfigurationProvider.cs

```diff
@@ -588,7 +589,7 @@ protected internal async Task<ConfigEnvironment> RemoteLoadAsync(string[] reques
                     if (response.StatusCode >= HttpStatusCode.BadRequest)
                     {
                         // HttpClientErrorException
-                        throw new HttpRequestException(message);
+                        throw new HttpRequestException(string.Format(statusMessage, response.StatusCode, WebUtility.UrlEncode(requestUri)));
```

Comment:
If you'd inline `statusMessage`, analyzers validate that the placeholders match the parameters. In the case of `LogInformation`, using `{Status}` and `{Path}` instead of `{0}` and `{1}` enables named parameters when using JSON logging. The `throw` line could use string interpolation instead, which is more efficient in .NET 6.